evo_researcher/benchmark/benchmark.py

```diff
@@ -222,9 +260,15 @@ def get_markets_summary(self) -> t.Dict[str, t.List[str]]:
         }
 
         for agent in [a.agent_name for a in self.registered_agents]:
+            agent_predictions = [self.get_prediction(agent_name=agent, question=q) for q in market_questions]
             markets_summary[f"{agent} p_yes"] = [
-                self.get_prediction(agent_name=agent, question=q).p_yes
-                for q in market_questions
+                (
+                    p.completion_prediction.p_yes if p.completion_prediction  # Probability
+                    else "N/A" if not p.question_evaluation  # Not evaluated for some reason
```

Comment:
Is there a chance that the question will not have been evaluated, but still answered? Might be good to have a sanity check here if we want to assert this is not possible.

Reply:
> Is there a chance that the question will not have been evaluated but still answered?

Not unless we introduce such a bug (or feature :-D). Added the check.